atomic_defi_design/qml/Components/DexModel.qml

```diff
@@ -0,0 +1,61 @@
+import QtQuick 2.15
+import QtQuick.LocalStorage 2.15
+
+ListModel {
+    id: control
+    property string dbName: "Test00000"
```

Comment:
it looks like this model not using atm, do you plan to utilize it later?

Reply:
I agree - this file need to be removed, on top of that the name `DexModel.qml` seems not appropriate for a storage model, something like `NotificationStorageModel.qml` would be more appropriate, if it's not used you can simply delete it